docs/goals.md

```diff
@@ -1,16 +1,53 @@
 # Goals and Non-Goals
 
-This document defines the purpose of the Message Format Working Group (MFWG)
-and informs the decisions about the scope and the priorities of its efforts.
+This document contains the charter for the MessageFormat Working Group (MFWG)
+and informs decisions about the scope and priority of its efforts.
 
-## Goals
+## Charter
 
-The primary task of the MFWG is to develop an industry standard for the
-representation of localizable dynamic message strings. A **_dynamic message
-string_** is a string whose content changes due to the value of or insertion
+A **_dynamic message string_** is a string whose content changes due to the value of or insertion
 of some data value or values.
 
-The design goals are listed below.
+The _Unicode MessageFormat Standard_ is an industry standard for the representation
+of localizable _dynamic message strings_.
+
+The MessageFormat Working Group maintains and extends the Unicode MessageFormat Standard,
+provides documentation;
+encourages implementation, including the development of tools and best practices;
+manages default and Unicode-defined function sets;
+and provides for interoperability with other standards.
+
+The MessageFormat Working Group is a working group the CLDR-TC.
+
+## Goals
+
+- Encourage adoption of Unicode MessageFormat as measured by developing [messageformat.unicode.org]
+  and other appropriate materials to include a user guide, MF1 migration guide,
+  implementation support, and usage materials while maintaining a high bar 
+  as a model for other Unicode websites.
+- Support a determination there are at least ### additional interoperable implementations.
```

Comment:
This seems more like a measure of how well we've succeeded at the preceding goal.
```suggestion
```

Reply:
The is a strawman for an implementation goal. Yes, it's a measure of how well our documentation has worked, but I envision a slightly more muscular approach to platforms teams (especially members)